src/NSwag.CodeGeneration.CSharp/Templates/Client.Class.QueryParameter.liquid

```diff
@@ -6,16 +6,16 @@ urlBuilder_.Append("{{ parameter.Name }}=").Append(System.Uri.EscapeDataString({
 foreach (var item_ in {{ parameter.VariableName }}) { urlBuilder_.Append("{{ parameter.Name }}=").Append(System.Uri.EscapeDataString(ConvertToString(item_, System.Globalization.CultureInfo.InvariantCulture))).Append("&"); }
 {% elseif parameter.IsDeepObject -%}
 {%     for property in parameter.PropertyNames -%}
-if ({{parameter.Name}}.{{property}} != null)
+if ({{parameter.Name}}.{{property.PropertyName}} != null)
```

Comment:
I think we should rename PropertyName to Name (same as in the parameter object on the left)